docs/config.json

```diff
@@ -1455,6 +1455,11 @@
       "destination": "/admin-guides/access-controls/access-lists/guide/",
       "permanent": true
     },
+    {
+      "source": "/access-controls/access-lists/nested-access-lists/",
+      "destination": "/admin-guides/access-controls/access-lists/nested-access-lists/",
+      "permanent": true
+    },
```

Comment:
I don't think you need a redirect, you need to add a new section above to the access control section.

Reply:
We generate the `admin-guides` section of the sidebar automatically from the directory now, so it's no longer necessary to add a section to `config.json`.

Reply:
Hm, it already shows up for me under that section? I'll take a look though...

Reply:
Ah I didn't know we generic it automatically now. Cool.